src/component/mxgraph/shape/event-shapes.ts

```diff
@@ -81,24 +81,21 @@ abstract class EventShape extends mxEllipse {
     super.paintVertexShape(c, x, y, w, h);
   }
 
-  protected paintInnerShape(paintParameter: PaintParameter): void {
+  private paintInnerShape(paintParameter: PaintParameter): void {
     const paintIcon = this.iconPainters.get(StyleUtils.getBpmnEventKind(this.style)) || (() => this.iconPainter.paintEmptyIcon());
     paintIcon(paintParameter);
   }
 
-  protected setDashedOuterShapePattern(paintParameter: PaintParameter, isInterrupting: string): void {
+  private static setDashedOuterShapePattern(paintParameter: PaintParameter, isInterrupting: string): void {
```

Comment:
I don't understand why this function is static now.

Reply:
as part of this PR, I realized that some methods were `protected` whereas there aren't overridden by subclasses. As we haven''t decide yet if they should be part of the public API for extension (see #593), I made them `private` for simplicity.
I did that for methods related to the canvas configurations. This one was initially untouched by this PR but is related to canvas configuration so I also passed it as private for consistency.
Once made private, as the method implementation doesn't depend on instance properties, my IDE suggested me to make them `static`.